controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -443,6 +448,158 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
     return true;
 }
 
+bool master_thread::handle_cmdu_1905_channel_preference_report_message(
+    Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    LOG(INFO) << "received CHANNEL_PREFERENCE_REPORT_MESSAGE";
+
+    // TODO: in actual channel selection task, it is important to validate that rx mid is identical
```

Comment:
I prefer not. It can cause more problems, than can actually help (actually I don't see how it helps at all). Problems such as what will happen if there will be several request from bml simultaneous. 
Checking the mid requires the channel selection task management.

Reply:
OK, I guess, since we are aiming for controller certification and I don't think the testbed will test that we are dropping replies with wrong MIDs.